docs/faq/terminology.rst

```diff
@@ -0,0 +1,33 @@
+.. _Observation_Terminology:
+
+***************
+Terminology
+***************
+
+Scheduling
+==================
+Determined during the python in astronomy conference:
```

Comment:
You should add just a bit more background here.  Something along the lines of:

Scheduling observations is a task with a lot of history but not a lot of formal terminology.  Here we define the terminology used in astroplan for scheduling-related tasks.  None of this is meant to be particularly official in any manner other than we try to use these terms consistently in the code and documentation of astroplan.